contentcuration/contentcuration/frontend/preview/views/Details.vue

```diff
@@ -0,0 +1,365 @@
+<template>
+
+  <div>
+    <LoadingText v-if="loading" />
+    <div v-else-if="hasDetails">
+      <DetailsRow
+        :label="$tr('creationHeading')"
+        :text="createdDate"
+      />
+      <DetailsRow
+        :label="$tr('sizeHeading')"
+        :text="sizeText"
+      />
+      <DetailsRow :label="$tr('resourceHeading')">
+        <template v-slot>
+          <p>{{ $formatNumber(details.resource_count) }}</p>
+          <!-- Using a table here instead of a list as the counts are better aligned -->
+          <VDataTable
+            v-if="kindCount.length"
+            :items="kindCount"
+            hide-actions
+            hide-headers
+          >
+            <template v-slot:items="props">
+              <td>
+                <ContentNodeIcon :kind="props.item.kind_id" />
+                <span>{{ translateConstant(props.item.kind_id) }}</span>
+              </td>
+              <td>{{ $formatNumber(props.item.count) }}</td>
+              <td v-if="$vuetify.breakpoint.smAndUp"></td>
+              <td v-if="$vuetify.breakpoint.mdAndUp"></td>
+              <td v-if="$vuetify.breakpoint.lgAndUp"></td>
+              <td v-if="$vuetify.breakpoint.xlAndUp"></td>
+            </template>
+          </VDataTable>
+        </template>
+      </DetailsRow>
+      <DetailsRow :label="$tr('containsHeading')">
+        <template v-slot>
+          <VChip v-if="details.includes.coach_content" color="grey lighten-2">
+            {{ $tr('coachHeading') }}
+          </VChip>
+          <VChip v-if="details.includes.exercises" color="grey lighten-2">
+            {{ $tr('assessmentsIncludedText') }}
+          </VChip>
+        </template>
+      </DetailsRow>
+      <DetailsRow
+        :label="$tr('coachHeading')"
+        :text="(details.includes.coach_content || 0).toString()"
```

Comment:
This should be wrapped in `$formatNumber` to ensure that the numeral gets properly rendered for the locale.